components/brave_new_tab_ui/components/default/widget/widgetMenu.tsx

```diff
@@ -26,6 +29,9 @@ interface Props {
   onLearnMore?: () => void
   onDisconnect?: () => void
   onRefreshData?: () => void
+  onAddSite?: () => void
+  customLinksEnabled?: boolean
+  onToggleCustomLinksEnabled?: () => void
```

Comment:
note that for follow-up we need to refactor everything already here, so that it works like a menu and accepts child items, instead of each menu item handling every possible option 😆